app/src/main/java/com/lagradost/cloudstream3/ui/library/LibraryFragment.kt

```diff
@@ -147,9 +147,15 @@ class LibraryFragment : Fragment() {
         activity?.theme?.resolveAttribute(android.R.attr.textColor, searchExitIconColor, true)
         searchExitIcon?.setColorFilter(searchExitIconColor.data)
 
+        // Declare a Handler and a Runnable
+        val inputDelayHandler = Handler(Looper.getMainLooper())
```

Comment:
All views are effectively handlers, so you can do stuff like binding?.mainSearch?.postDelayed()